yaptide/persistence/models.py

```diff
@@ -1,15 +1,30 @@
 from yaptide.persistence.database import db
+from werkzeug.security import generate_password_hash, check_password_hash
 
 ############### Example user ###############
 # (this is an example model, demonstration pourpose only)
 
 
-class ExampleUserModel(db.Model):
+class UserModel(db.Model):
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String, nullable=False)
+    email = db.Column(db.String, nullable=False)
```

Comment:
replace `email` with `login_name`
login_name could be an email (like grzanka@agh.edu.pl) or plain login (like grzanka)

Reply:
changed